### DISCUSSION TITLE
Sorting bug in the page editor? 🤔

### DISCUSSION THREAD

**Discussion by 4rt10m** [directus/directus#16129] [Archive]
I have a Sections M2A field (which links to all other components I’ve built). I have created the sort field and it works flawlessly. Until, I add more than 15 items and it starts paginating 🤔

As soon as it goes to page 2, I can no longer use the sort functionality. So from here, have 2 questions. 

1. Can I increase the number of items from 15 to say 30? 
2. Is it a bug, or I messed it up somehow?

![9D3B63F9-404A-465C-9F86-0D5126E7F6DF](https://user-images.githubusercontent.com/21058332/197409923-5e412d88-01c4-4bed-9f43-1bdb44543840.jpeg)
![6796C206-AC5A-4703-B886-1D85D4E9AB23](https://user-images.githubusercontent.com/21058332/197409924-b33933e0-61be-4cb4-956b-274208bf1c82.jpeg)

**Comment by azrikahar** [comment]
> 1. Can I increase the number of items from 15 to say 30? 

Yes, you can increase it by changing the "Per Page" option in M2A (or other relational fields) which currently defaults to 15 👍

![](https://user-images.githubusercontent.com/42867097/197431589-1cded01b-9987-49c9-95b9-60403d394ab6.png)

> 2. Is it a bug, or I messed it up somehow?

It is currently a known limitation rather than a bug (refer to this comment:  https://github.com/directus/directus/discussions/12818#discussioncomment-2585113), and it is being discussed in #12818.

**Reply by 4rt10m** [reply]
@azrikahar thank you! I thought I've read through all the discussions but clearly missed this one!

Thanks a lot, was driving me crazy but makes sense now! 

Good to know it's a known thing and easy solution in my case!

**Reply by bee8bit** [reply]
Addendum: I just noticed that setting _Items Per Page_ to -1 does disable pagination ... but disables manual sorting as well.
